cmd/server/initialsForServer.go

```diff
@@ -79,24 +87,37 @@ func foa4Server() error {
 	if _, exists := os.LookupEnv("DATABASE_DSN"); !exists {
 		dbEndPoint = dbFlag
 	}
+	memStor = &MemStorage{
+		Gaugemetr: make(map[string]gauge),
+		Countmetr: make(map[string]counter),
+		Mutter:    &mtx,
+	}
 	if dbEndPoint == "" {
 		log.Println("No base in Env variable and command line argument")
+		inter = memStor // если базы нет, подключаем in memory Storage
 		return nil
 	}
-	ctx := context.Background()
-	mb, err := pgx.Connect(ctx, dbEndPoint)
-	MetricBaseStruct = dbaser.Struct4db{MetricBase: mb, Ctx: ctx, IsBase: false}
+	dbStorage = &basis.DBstruct{}
```

Comment:
```suggestion
	dbStorage := &basis.DBstruct{}
```
Нет смысла делать эту переменную глобальной. Лучше вообще в БД где-нибудь поместить конструктор, который на вход будет получать конфигурацию (или отдельно строку подключения), цепляться к базе, набрасывать миграцию и возвращать либо инициализированный объект `*basis.DBstruct`, либо ошибку. И так для всех стораджей. Буду рад увидеть это в следующей итерации.